### ISSUE TITLE
Inconsistent App Switching: Multiple Attempts Required to Open Apps from Favourites

### ISSUE THREAD

**Issue by ashuGulati** [8bitPit/Niagara-Issues#2880]
### Describe the bug

When attempting to switch from the currently open 'Vocabulary' app to the 'Search' app via the favourites menu, it often requires multiple attempts (observed three times in a recent case) for the 'Search' app to open. This behaviour is inconsistent, as it does not occur every time but has been noticed frequently.

### Steps to reproduce

1. Open the 'Vocabulary' app.
2. Attempt to switch to the 'Search' app using the favourites menu.
3. Observe whether the 'Search' app opens on the first attempt or if multiple attempts are required.

### Does the issue also occur with other 3rd-party launchers?

Problem occurs with other launchers

### Device name

Samsung Galaxy M31s

### Android version

Android 12

### Screenshots / Screen recordings

https://github.com/user-attachments/assets/0b7edb5d-9a76-48c2-8c95-c859b714063c

**Comment by NiagaraSupport** [comment]
Hey there, thanks for the report.
Does this also happen on other 3rd Party Launchers?

**Comment by ashuGulati** [comment]
Hey, I do not know about other 3rd party launchers. Do let me know which other launcher I should test this bug with. Thanks.

**Comment by contact-maximux** [comment]
why your widget feedly is empty. may be height issue

**Comment by ashuGulati** [comment]
> why your widget feedly is empty. may be height issue

Good catch! That's another issue. Not exactly because of the height but it randomly disappears on the news and feedly displays it as empty.

**Comment by NiagaraSupport** [comment]
> Hey, I do not know about other 3rd party launchers. Do let me know which other launcher I should test this bug with. Thanks.

Thank you very much. I'd suggest trying Nova Launcher. This helps us filter out potential hardware, OS, or third-party issues, that we don't have much control over.

**Comment by ashuGulati** [comment]
> > Hey, I do not know about other 3rd party launchers. Do let me know which other launcher I should test this bug with. Thanks.
> 
> Thank you very much. I'd suggest trying Nova Launcher. This helps us filter out potential hardware, OS, or third-party issues, that we don't have much control over.

I tried Nova Launcher but I don't see this issue occurring there.

**Comment by NiagaraSupport** [comment]
Thanks for testing it. We'll get back to you once we know more.
